smack-extensions/src/test/java/org/jivesoftware/smackx/pubsub/PubSubNodeTest.java

```diff
@@ -0,0 +1,86 @@
+/**
+ *
+ * Copyright 2018 Timothy Pitt
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jivesoftware.smackx.pubsub;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+import org.jivesoftware.smack.SmackException;
+import org.jivesoftware.smack.ThreadedDummyConnection;
+import org.jivesoftware.smack.XMPPException;
+import org.jivesoftware.smack.test.util.TestUtils;
+import org.jivesoftware.smack.util.PacketParserUtils;
+import org.jivesoftware.smackx.pubsub.packet.PubSub;
+
+import org.junit.Test;
+import org.jxmpp.jid.DomainBareJid;
+import org.jxmpp.jid.impl.JidCreate;
+import org.jxmpp.stringprep.XmppStringprepException;
+import org.xmlpull.v1.XmlPullParser;
+
+public class PubSubNodeTest
+{
+
+    public static final DomainBareJid DUMMY_PUBSUB_SERVICE;
+
+    static {
+        DomainBareJid pubSubService;
+        try {
+            pubSubService = JidCreate.domainBareFrom("pubsub.dummy.org");
```

Comment:
There exists `org.jxmpp.jid.JidTestUtil.PUBSUB_EXAMPLE_ORG` for just this reason. Please use that instead.